cluster/addons/dns-horizontal-autoscaler/dns-horizontal-autoscaler.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       containers:
       - name: autoscaler
-        image: gcr.io/google_containers/cluster-proportional-autoscaler-amd64:1.0.0
+        image: gcr.io/google_containers/cluster-proportional-autoscaler-amd64:1.1.0-r2
```

Comment:
Sorry, forgot to mention, please remove `--mode=linear` flag for the autoscaler container. I aggressively deprecated this flag in 1.1.